### DISCUSSION TITLE
S3 and flysystem package are auto removed when install pimcore/data-hub package.

### DISCUSSION THREAD

**Discussion by ravi-vegda** [pimcore/pimcore#9752] [Q&A / Support]
I have used S3 package to integrate AWS S3 in pimcore.
composer require league/flysystem-aws-s3-v3

but whenever I install Pimcore data hub package (composer require pimcore/data-hub), process will remove S3 and flysystem package.

![image_2021_07_13T15_18_06_935Z](https://user-images.githubusercontent.com/86904617/125484234-03f1e040-5a41-4ee8-b0ac-c97061858fa0.png)

**Comment by jdreesen** [comment]
Are you sure `league/flysystem-aws-s3-v3` is still present in your `composer.json` before installing `pimcore/data-hub`? If not, it'll get removed because composer thinks it's no longer needed.

**Comment by langert** [comment]
could you solve the problem?
I got amazon s3 storage running, but installing data-import shows up a conflict of versions with flysystem => phpseclib.

![grafik](https://user-images.githubusercontent.com/11093410/132340221-390beb28-20a0-4afd-ae88-ce2fb470afc6.png)
